crates/containerd-shim-wasm/src/sandbox/cli.rs

```diff
@@ -48,7 +86,17 @@ pub fn shim_main<'a, I>(
     I: 'static + Instance + Sync + Send,
     I::Engine: Default,
 {
+    if cfg!(feature = "opentelemetry") {
+        // read TRACECONTEXT env var that's set by the parent process
+        if let Ok(ctx) = std::env::var("TRACECONTEXT") {
```

Comment:
NOTE: It uses `TRACECONTEXT` env var to pass tracing context from the parent shim process to the child process.